cs-property-service/src/main/resources/db/migration/main/V20200625110941__cs_duplicatecopy_property_ddl.sql

```diff
@@ -0,0 +1,142 @@
+DROP TABLE IF EXISTS cs_pt_duplicatecopy_property_audit_v1;
+DROP TABLE IF EXISTS cs_pt_duplicatecopy_propertydetails_audit_v1;
+DROP TABLE IF EXISTS cs_pt_duplicatecopy_applicant_audit_v1;
+
+--> Duplicate copy Property table
+
+CREATE TABLE cs_pt_duplicatecopy_property_v1 (
+   id           			CHARACTER VARYING (256) NOT NULL,
+   transit_number       	CHARACTER VARYING (256) NOT NULL,
+   tenantid			    	CHARACTER VARYING (256),
+   applicationType			CHARACTER VARYING  (64),
+   duplicatecopy_state    	CHARACTER VARYING (256),
+   duplicatecopy_action   	CHARACTER VARYING (256),
+  
+   created_by           	CHARACTER VARYING (128) NOT NULL,
+   created_time         	bigint,
+   modified_by     			CHARACTER VARYING (128),
+   modified_time       		bigint,
+
+  CONSTRAINT pk_cs_pt_duplicatecopy_property_v1 PRIMARY KEY (id),
+  CONSTRAINT uk_cs_pt_duplicatecopy_property_v1 UNIQUE (transit_number)
+);
+
+--> Duplicate copy Property details table
+CREATE TABLE cs_pt_duplicatecopy_propertydetails_v1 (
+   id           		CHARACTER VARYING (256) NOT NULL,
+   property_id       	CHARACTER VARYING (256),
+   transit_number       CHARACTER VARYING (256) NOT NULL,
+   tenantid			    CHARACTER VARYING (256),
+   sector           	CHARACTER VARYING (256),
+   additional_details   CHARACTER VARYING (256),
+  
+   created_by           CHARACTER VARYING (128) NOT NULL,
+   created_time         bigint,
+   modified_by     		CHARACTER VARYING (128),
+   modified_time       	bigint,
+
+  CONSTRAINT pk_cs_pt_duplicatecopy_propertydetails_v1 PRIMARY KEY (id),
+  CONSTRAINT uk_cs_pt_duplicatecopy_propertydetails_v1 UNIQUE (transit_number),
+  CONSTRAINT fk_cs_pt_duplicatecopy_propertydetails_v1 FOREIGN KEY (property_id) REFERENCES cs_pt_duplicatecopy_property_v1 (id)
+  ON UPDATE CASCADE
+  ON DELETE CASCADE
+);
+
+--> Duplicate copy applicant details table
+CREATE TABLE cs_pt_duplicatecopy_applicant_v1 (
+   id           		CHARACTER VARYING (256) NOT NULL,
+   property_id       	CHARACTER VARYING (256),
+   owner_id       		CHARACTER VARYING (256),
+   tenantid			    CHARACTER VARYING (256),
+   name       			CHARACTER VARYING (256),
+   email       			CHARACTER VARYING (256),
+   phone       			CHARACTER VARYING (256),
+   guardian				CHARACTER VARYING (256),
+   address				CHARACTER VARYING (256),
+   application_status   CHARACTER VARYING (256),
+   active_state         CHARACTER VARYING (256),
+   is_primary_owner   	CHARACTER VARYING (256),
+  
+   created_by           CHARACTER VARYING (128) NOT NULL,
+   created_time         bigint,
+   modified_by     		CHARACTER VARYING (128),
+   modified_time       	bigint,
+
+  CONSTRAINT pk_cs_pt_duplicatecopy_applicant_v1 PRIMARY KEY (id),
+  CONSTRAINT fk_cs_pt_duplicatecopy_applicant_v1 FOREIGN KEY (property_id) REFERENCES cs_pt_duplicatecopy_property_v1 (id)
+  ON UPDATE CASCADE
+  ON DELETE CASCADE
+);
+
+--> Duplicate copy document table
+CREATE TABLE cs_pt_duplicate_ownership_application(
```

Comment:
We will not need this table. Any property details can be obtained using the foreign key.